packages/start-server-core/tests/mocks/router-entry.ts

```
@@ -0,0 +1,30 @@
+import { AnyRouter } from '@tanstack/router-core'
+
+export let currentHandlers: Record<string, any> = {}
```

ESLint

[error] 'currentHandlers' is never reassigned. Use 'const' instead.

(prefer-const)